tower-http/src/compression_utils.rs

```diff
@@ -352,6 +428,7 @@ pub enum CompressionLevel {
     Precise(i32),
 }
 
+#[allow(clippy::derivable_impls)]
```

Comment:
Why not derive this? Clippy should take the MSRV into account, indeed deriving default on an enum (where the default variant has no fields) requires only [Rust 1.62](https://blog.rust-lang.org/2022/06/30/Rust-1.62.0.html#default-enum-variants).